packages/amplify-e2e-tests/src/__tests__/storage.test.ts

```diff
@@ -1,7 +1,10 @@
+import { JSONUtilities } from 'amplify-cli-core';
```

Comment:
if possible, we should rebase against the main branch before making these changes. The storage tests have been split up into 3 files.